examples/44_cloudflare/wrangler.toml

```diff
@@ -2,7 +2,7 @@
 # See https://developers.cloudflare.com/workers/wrangler/configuration/
 name = "waku-project"
 compatibility_date = "2024-09-23"
-compatibility_flags = [ "nodejs_als" ]
+compatibility_flags = [ "nodejs_compat" ]
```

Comment:
I would like to make sure it works with `nodejs_als` only. Can you avoid `readFile` and read just test from js file instead?

Reply:
Updated!